### DISCUSSION TITLE
Discord'a katılma

### DISCUSSION THREAD

**Discussion by okazkayasi** [acikkaynak/eczane-frontend#116] [General]
@volkantash tarafindan acilmistir. Mesaji asagida: 

Selamun aleyküm.

Discord sunucularına katılmak için şöyle örkler/linkler oluşturabiliriz.

https:// url / discord

Bunu doğrudan katılma örkü/linki olarak kullanabiliriz. Olmadı katılmak için bir sayfaya dönüştürebiliriz.

**Comment by volkantash** [comment]
Sızı/issue olarak başlatılan bir konuyu sağ alt yanda bulunan seçek/menu içindeki "**Convert to discussion**" (Tartışmaya dönüştür) yazısına tıklayıp tartışma konusuna dönüştürebilirsiniz. @okazkayasi  
![resim](https://user-images.githubusercontent.com/25375593/218767936-e6e8f780-37b9-4379-ad15-8d19969f50fd.png)

**Reply by okazkayasi** [reply]
harika tesekkurler
